kitsune/community/cron.py

```diff
@@ -0,0 +1,80 @@
+import cronjobs
+from datetime import datetime, timedelta
+
+from django.conf import settings
+from django.contrib.auth.models import User
+from django.contrib.sites.models import Site
+from django.db.models import F, Q
+
+from tower import ugettext as _
+
+from kitsune.questions.models import Answer
+from kitsune.sumo.email_utils import make_mail, safe_translation, send_messages
+from kitsune.users.models import Profile
+from kitsune.wiki.models import Revision
+
+
+@cronjobs.register
+def send_welcome_emails():
+    """Send a welcome email to first time contributors.
+
+    Anyone who has made a contribution more than 24 hours ago and has not
+    already gotten a welcome email should get a welcome email.
+    """
+
+    wait_period = datetime.now() - timedelta(hours=24)
```

Comment:
Since the cronjob runs once an hour, I think this will sometimes result in things being processed that actually happened 23 hours ago, or actually happened 25 hours ago. I think that is fine. I don't think it will blow up or make anyone sad. Sound accurate?